src/Catel.MVVM/Catel.MVVM.Shared/MVVM/ViewModels/ViewModelManager.cs

```diff
@@ -101,9 +130,14 @@ public IEnumerable<IViewModel> ActiveViewModels
         {
             get
             {
-                lock (_managedViewModelsLock)
+                _managedViewModelsLock.EnterReadLock();
```

Comment:
It is have to be instance of some class (because it have to keep ReaderWritterLockSlim instance) so it won't be helper in sensus stricte.